app/assets/stylesheets/style.scss

```diff
@@ -318,6 +318,11 @@ label small {
   }
 }
 
+.container{
```

Comment:
modifying the entire `container` class might affect other parts of MK, doesn't it? Couldn't you use ids or use the previously defined class?

Reply:
I used an id. Thanks for the suggestion